miden-lib/asm/miden/kernels/tx/prologue.masm

```diff
@@ -495,56 +495,64 @@ end
 #! - authenticate the note and its metadata in the note Merkle tree from the block the note was
 #!   created in.
 #!
-#! Operand stack: [AUTH_DIGEST]
-#! Advice stack: [leaf_pos, SUB_HASH, NOTE_ROOT, note_index]
+#! Operand stack: [NOTE_HASH]
+#! Advice stack: [leaf_pos, BLOCK_SUB_HASH, NOTE_ROOT, note_index]
 #! Output: []
 #!
 #! Where:
-#! - AUTH_DIGEST is the digest of the input note data computed as hash(NOTE_HASH, NOTE_METADATA).
+#! - NOTE_HASH is the digest of the input note data computed as hash(NOTE_HASH, NOTE_METADATA).
```

Comment:
The definition should be `hash(NOTE_ID, NOTE_METADATA)` rather than `hash(NOTE_HASH, NOTE_METADATA)`.

Reply:
updated